src/models/student.py

```diff
@@ -11,6 +12,87 @@ class Student(db.Model):
     name = db.Column(db.String(200))
     created_at = db.Column(db.DateTime, default=datetime.datetime.utcnow)
 
+    @staticmethod
+    def generate_enrolment_key(mobile, from_helpline):
+        '''
+            this function helps to create a a record of a new student in the database were the from_helpine varibale helps to track was the call from helpline or was manually created and send the enrollment key to the mobile number
+
+            if was from_helpline record the incoming calls also
+            
+            USAGE :
+                Student.generate_enrolment_key(mobile, from_helpline)
+
+            params: 
+                mobile : String required
+                from_helpline : Boolean required
+
+        '''
+        student = Student(stage=app.config['STAGES']['EKG'])
+        db.session.add(student)
+        db.session.commit()
```

Comment:
Are you sure you will have to commit this? If you commit at the end won't SQLAlchemy take care of it on its own?

Reply:
I have tried it but on the adding student.id to student_contact adds null in the column due to which I had to use commit every time is there any better solution to this?